graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -137,19 +138,21 @@
 VisitorReturnType = Union[Node, VisitorAction]
 
 
-class RenamingMapping(Protocol):
+class TypeRenamingMapping(Protocol):
     def get(self, key: str, default: Optional[str]) -> Optional[str]:
-        """Define mapping for renaming object."""
+        """Define mapping for type_renamings object."""
         ...
 
 
-def rename_schema(schema_ast: DocumentNode, renamings: RenamingMapping) -> RenamedSchemaDescriptor:
-    """Create a RenamedSchemaDescriptor; rename/suppress types and root type fields using renamings.
+def rename_schema(
+    schema_ast: DocumentNode, type_renamings: TypeRenamingMapping
+) -> RenamedSchemaDescriptor:
+    """Create a RenamedSchemaDescriptor; rename/suppress types and root type fields.
 
-    Any type, interface, enum, or fields of the root type/query type whose name
-    appears in renamings will be renamed to the corresponding value if the value is not None. If the
-    value is None, it will be suppressed in the renamed schema and queries will not be able to
-    access it.
+    Any object type, interface type, enum type, or field of the root type/query type has a name. Let
+    the name be called type_name. If type_renamings.get(type_name, _) is not None, the type or field
```

Comment:
Confused about the `_` in `type_renamings.get(type_name, _)`. Is that supposed to be `None` instead?

Reply:
The idea here is that either the type named `type_name` gets suppressed or not, and whether or not it gets suppressed doesn't depend on the default return value but rather only on `type_name`.

`type_renamings.get(type_name, type_name)` or `type_renamings.get(type_name, None)` would both give a correct sentence here, despite the former leaving types alone by default and the latter suppressing them by default; in either case, if the return isn't None, then the type/ query type field gets renamed to whatever gets returned.

Reply:
Ok, makes sense to me-- one clarification:

* This sentence (which occurs in the Github reply) is true: `If type_renamings.get(type_name, None) is not None, then the type or root query field will be renamed to whatever is returned`

* This sentence (which occurs in the comment in the code) is also true: `Any such names that do not appear in renamings will be unchanged`

* The part that I worded confusingly is my original Github reply, because it brings up a scenario that doesn't occur in the code (default `type_renamings` return value being `None`) in order to show that any default return value would make the comment true, even if that return value is `None`. I'll change it back, since it's misleading